src/backend/distributed/commands/cluster.c

```diff
@@ -40,9 +40,12 @@ PreprocessClusterStmt(Node *node, const char *clusterCommand,
 
 	if (clusterStmt->relation == NULL)
 	{
-		ereport(WARNING, (errmsg("not propagating CLUSTER command to worker nodes"),
-						  errhint("Provide a specific table in order to CLUSTER "
-								  "distributed tables.")));
+		if (!DisablePreconditions)
```

Comment:
maybe `EnablePropagationWarnings`?

(and use separate GUCs for other purposes)

Reply:
Added more GUCs for different tasks related to vanilla testing. And renamed the old GUC.